webapp/utils/constants.jsx

```diff
@@ -59,7 +59,8 @@ export const Preferences = {
     EMAIL_INTERVAL: 'email_interval',
     INTERVAL_IMMEDIATE: 30, // "immediate" is a 30 second interval
     INTERVAL_FIFTEEN_MINUTES: 15 * 60,
-    INTERVAL_HOUR: 60 * 60
+    INTERVAL_HOUR: 60 * 60,
+    INTERVAL_NEVER: 'never'
```

Comment:
Otherwise, this looks really good. I like moving all that logic into a single place